irc/connection.py

```diff
@@ -75,8 +75,6 @@ class AioFactory:
 
     """
 
-    family = socket.AF_INET
```

Comment:
What's this change about? I'm pretty sure this property is referenced in `Factory.connect`.

Reply:
Sorry, I probably should have done that as a separate PR, but I was just doing some cleanup while I was looking at the testing bug.  It is indeed referenced in `Factory.connect`, and the property is still there in the `Factory` class.  The line flagged above is instead removing it from the `AioFactory`, where it is **not** currently used (it was leftover from when the `Factory` class was copied as the base of `AioFactory`).  I noticed the orphaned property when investigating the test bug and removed it -- although, now that I look at it, the doc string for `AioFactory` is _also_ just a copypasta from `Factory` and should be updated accordingly.

Reply:
Oh, thanks for the clarification. Between the GitHub UI and my own local copy without the change, I'd missed where the duplication was.